include/psa/crypto.h

```diff
@@ -2467,6 +2466,12 @@ psa_status_t psa_aead_set_nonce(psa_aead_operation_t *operation,
  * If this function returns an error status, the operation enters an error
  * state and must be aborted by calling psa_aead_abort().
  *
+ * \deprecated
+ * You may call this function after calling psa_aead_set_nonce() or
+ * psa_aead_generate_nonce() (but before calling any update or finish function).
+ * This possibility is deprecated and will be removed in a future version of
+ * the library.
```

Comment:
It would be clearer to invert the language and talk about what will be required in future versions of the library:
```suggestion
 * This behaviour is deprecated. In future versions of the library, this
 * function will need to be called BEFORE calling psa_aead_generate_nonce()
 * or psa_aead_set_nonce().
```